src/pages/gateway/create-mesh.md

```diff
@@ -189,6 +189,36 @@ aio api-mesh:update update-mesh.json
     }
 ```
 
+## Modify projects and workspaces
+
+<InlineAlert variant="info" slots="text"/>
+
+When creating a mesh for the first time, you must select the project and workspace that you want to create the mesh in. Alternatively, you can use the following [aio commands](https://github.com/adobe/aio-cli#commands) to select or deselect a project or workspace without interacting with your mesh.
+
+[aio commands](https://github.com/adobe/aio-cli#commands) allow you to list the current selections, manually select a project or workspace, or remove the currently selected project or workspace from the cache.
+
+### View cached project and workspace
+
+To see your current cache configuration, use the [`aio config:get:console`](https://github.com/adobe/aio-cli#aio-configget-key) command, which includes the currently selected organization, project, and workspace.
+
+### Select a project or workspace
+
+If you want to change the selected project, use the [`aio console:project:sel`](https://github.com/adobe/aio-cli#aio-consoleprojectsel-projectidorname) or [`aio console:project:select`](https://github.com/adobe/aio-cli#aio-consoleprojectselect-projectidorname) commands followed by the name or id of the project. You will be prompted to select a project.
+
+If you want to change the selected workspace, use the [`aio console:workspace:sel`](https://github.com/adobe/aio-cli#aio-consoleworkspacesel-workspaceidorname) or [`aio console:workspace:select`](https://github.com/adobe/aio-cli#aio-consoleworkspaceselect-workspaceidorname) commands followed by the name or id of the workspace. You will be prompted to select a workspace.
+
+By [default](https://developer.adobe.com/app-builder/docs/getting_started/first_app/#2-creating-a-new-project-on-developer-console) projects have `prod` and `stage` workspace.
```

Comment:
This sentence is out of place as written. Either it to the top or delete it entirely. I'm not sure if all projects always have those workspaces.